scripts/push-broker.sh

```diff
@@ -80,6 +80,11 @@ if [[ -z "$GSB_SERVICE_CSB_AWS_S3_BUCKET_PLANS" ]]; then
 fi
 echo "    GSB_SERVICE_CSB_AWS_S3_BUCKET_PLANS: $(echo "$GSB_SERVICE_CSB_AWS_S3_BUCKET_PLANS" | jq @json)" >>$cfmf
 
+if [[ -z "$GSB_SERVICE_CSB_AWS_POSTGRES_PLANS" ]]; then
+  GSB_SERVICE_CSB_AWS_POSTGRES_PLANS='[{"name":"default","id":"de7dbcee-1c8d-11ed-9904-5f435c1e2316","description":"Default Postgres plan","display_name":"default","instance_class": "db.m6i.large","postgres_version": "14.2","storage_gb": 10}]'
+fi
+echo "    GSB_SERVICE_CSB_AWS_S3_BUCKET_PLANS: $(echo "$GSB_SERVICE_CSB_AWS_S3_BUCKET_PLANS" | jq @json)" >>$cfmf
```

Comment:
**question** Is this supposed to be `GSB_SERVICE_CSB_AWS_POSTGRES_PLANS`?

Reply:
Great catch! Fixed that

Reply:
I think the `echo` line here needs to echo the `GSB_SERVICE_CSB_AWS_POSTGRESQL_PLANS` variable?